src/libexpr/common-eval-args.hh

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include "args.hh"
+#include "flake/flakeref.hh"
```

Comment:
This `#include` is unnecessary here.

Reply:
I can take the import away here and add it to `nix/command.hh`, where it's needed to define the registry and lockfile overrides in `EvalCommand`.